DSCResources/MSFT_xWebsite/MSFT_xWebsite.schema.mof

```diff
@@ -10,17 +10,17 @@ Class MSFT_xWebBindingInformation
 };
 
 
-[ClassVersion("2.0.0"), FriendlyName("xWebsite")] 
+[ClassVersion("2.0.0"), FriendlyName("xWebsite")]
 class MSFT_xWebsite : OMI_BaseResource
 {
   [Key] string Name;
-  [write] string PhysicalPath;
+  [Required] string PhysicalPath;
```

Comment:
Fortunately most of the code I wrote was in the tests to validate the changes I made didn't break everything so its not a total loss to change it. Do we just want to change it back to write, and throw in the "right places" if we don't have the data? Not as clean but it'd handle all cases.